src/openforms/utils/json_logic/descriptions.py

```diff
@@ -319,7 +319,19 @@ def op_today(*args, **kwargs) -> str:
 
 op_date = FunctionLike(_("date({args})"))
 op_datetime = FunctionLike(_("datetime({args})"))
-op_rdelta = TemplatedArgs(
-    _("{years} year(s), {months} month(s), {days} day(s)"),
-    arg_names=["years", "months", "days"],
-)
+
+
+@add_boilerplate()
+def op_rdelta(*args):
+    if len(args) == 1:
+        return _("%(years)s year(s)") % {"years": args[0]}
```

Comment:
we prefer the `.format` syntax

```suggestion
        return _("{years} year(s)").format(years=args[0])
```